gym/utils/option_parser.py

```diff
@@ -0,0 +1,53 @@
+"""
+Utility functions used for classic control environments.
+"""
+
+from typing import Optional, Union
+import numpy as np
+
+
+def verify_number(x: Union[int, float, np.ndarray]) -> bool:
```

Comment:
unfortunately no:
`np.isscalar(3) = True` (good)
`np.isscalar(np.array(3)) = False`  (bad, we want it to be True)
`np.isscalar('string') = True` (bad, we want it to be False)

Reply:
Hm, good point. How about using `isinstance(x, numbers.Number)` instead, as suggested by [official numpy docs](https://numpy.org/doc/stable/reference/generated/numpy.isscalar.html)? This would return `False` for `np.array(3)`, but I think that's perfectly fine. Or are there situations where supporting zero-dim arrays is actually necessary?

My overall issue here is that I'm hesitant to roll a custom `is_number` function which feels like something common enough that there definitely is an existing "best-practices" implementation

Reply:
in the previous PR it was suggested that some users might pass in numpy (zero-dim) arrays, which is why i switched to this method:
https://github.com/openai/gym/pull/2921#discussion_r905455223

i agree it'd be nice to have a more standarized method for checking `is_number`, but so far most methods i've found fail at one of the use-cases described.